website/files/models/figshare.py

```diff
@@ -40,7 +40,7 @@ def update(self, revision, data, user=None):
             <div class="alert alert-info" role="alert">
             The file "{name}" is still a draft on figshare. <br>
             To view it  on the OSF
-            <a href="https://figshare.zendesk.com/hc/en-us/articles/203712033-How-do-I-publish-my-data-/">publish</a>
+            <a href="https://support.figshare.com/support/solutions ">publish</a>
```

Comment:
It shouldn't matter for functionality, but there's an extra space at the end of the `href` string. 

@sloria, can you fix this on merge?